src/rsa/padding.rs

```diff
@@ -116,8 +116,9 @@ pkcs1_digestinfo_prefix!(
     [ 0x60, 0x86, 0x48, 0x01, 0x65, 0x03, 0x04, 0x02, 0x03 ]);
 
 impl Verification for PKCS1 {
-    fn verify(&self, msg: untrusted::Input, encoded: untrusted::Input)
-              -> Result<(), error::Unspecified> {
+    fn verify(&self, msg: untrusted::Input, encoded: untrusted::Input,
+              bit_len: usize) -> Result<(), error::Unspecified> {
+        assert_eq!(encoded.len() * 8, bit_len);
```

Comment:
Is this assertion useful? It seems like we can just add a `_` prefix to the parameter name to show that it is unused so that we don't get an unused parameter warning.

Also, I think in some cases this assertion isn't correct? If it were correct then we wouldn't need the `bit_len` parameter at all, right?